<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnionWidget</class>
 <widget class="QWidget" name="UnionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>856</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background: white;
}
QLabel {
    color: #000000;
    font-size: 14px;
    background-color: transparent;
}
QLineEdit {
    padding: 8px 12px;
    border: 1px solid #ced4da;
    border-radius: 4px;
    background-color: #ffffff;
    font-size: 14px;
    min-height: 20px;
    color: #495057;
}
QLineEdit:focus {
    border: 1px solid #80bdff;
    outline: 0;
}
QPushButton {
    color: #495057;
    background-color: #e9ecef;
    border: 1px solid #ced4da;
    padding: 8px;
    border-radius: 4px;
    font-size: 14px;
}
QPushButton:hover {
    background-color: #dee2e6;
}
QPushButton#save_button {
    color: #fff;
    background-color: #007bff;
    border-color: #007bff;
    font-weight: bold;
}
QPushButton#save_button:hover {
    color: #fff;
    background-color: #0069d9;
    border-color: #0062cc;
}
QPushButton#save_button:pressed {
    background-color: #0062cc;
    border-color: #005cbf;
}
QComboBox {
    background-color: white;
    border: 1px solid #dee2e6;
    border-radius: 4px;
    padding: 5px 10px;
    min-height: 30px;
    color: #495057;
}
QComboBox:focus {
    border: 1px solid #339af0;
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: center right;
    width: 30px;
    border: none;
}
QComboBox::down-arrow {
    image: url(src/ui/resources/images/down-arrow-svgrepo-com.svg);
    width: 10px;
    height: 10px;
    margin-right: 1px;
}
QComboBox QAbstractItemView {
    background-color: white;
    border: 1px solid #dee2e6;
    border-radius: 4px;
    outline: none;
    selection-background-color: #e9ecef;
    selection-color: #000;
}
QComboBox QAbstractItemView::item {
    padding: 8px 20px;
    color: #495057;
}
QComboBox QAbstractItemView::item:selected {
    background-color: #e9ecef;
    color: #000;
}
QTableWidget {
    background-color: #f8f9fa;
    border: 1px solid #dee2e6;
    border-radius: 4px;
    gridline-color: #e9ecef;
    color: #212529;
}
QTableWidget::item {
    padding: 4px;
    border-bottom: 1px solid #e9ecef;
    color: #212529;
    background-color: white;
}
QTableWidget::item:selected {
    background-color: #e7f5ff;
    color: #212529;
}
QTableWidget::item:focus {
    outline: none;
    border: none;
    background: #e7f5ff;
    color: #212529;
}
QHeaderView::section {
    background-color: #f8f9fa;
    color: #212529;
    padding: 8px;
    border: none;
    border-right: 1px solid #dee2e6;
    border-bottom: 1px solid #dee2e6;
    font-weight: bold;
}
QScrollBar:vertical {
    border: none;
    background: #f8f9fa;
    width: 10px;
    margin: 0px;
}
QScrollBar::handle:vertical {
    background: #dee2e6;
    min-height: 20px;
    border-radius: 5px;
}
QScrollBar::handle:vertical:hover {
    background: #adb5bd;
}
QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    height: 0px;
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: none;
}
QScrollBar:horizontal {
    border: none;
    background: #f8f9fa;
    height: 10px;
    margin: 0px;
}
QScrollBar::handle:horizontal {
    background: #dee2e6;
    min-width: 20px;
    border-radius: 5px;
}
QScrollBar::handle:horizontal:hover {
    background: #adb5bd;
}
QScrollBar::add-line:horizontal, QScrollBar::sub-line:horizontal {
    width: 0px;
}
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
    background: none;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="title_label">
     <property name="styleSheet">
      <string notr="true">font-size: 20px;
font-weight: bold;
color: #000000;
background-color: transparent;</string>
     </property>
     <property name="text">
      <string>테이블 행 결합</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="leftLayout" stretch="0,1">
       <property name="sizePolicy" stdset="0">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="current_datasource_label">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>42</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>42</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size: 16px;
font-weight: bold;
color: #000000;
background-color: transparent;</string>
           </property>
           <property name="text">
            <string>현재 데이터 소스</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="current_datasource_table">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>300</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QTableWidget {
    background-color: #f8f9fa;
    border: 1px solid #dee2e6;
    border-radius: 4px;
    gridline-color: #e9ecef;
    color: #212529;
}
QTableWidget::item {
    padding: 4px;
    border-bottom: 1px solid #e9ecef;
    color: #212529;
    background-color: white;
}
QTableWidget::item:selected {
    background-color: #e7f5ff;
    color: #212529;
}
QTableWidget::item:focus {
    outline: none;
    border: none;
    background: #e7f5ff;
}
QHeaderView::section {
    background-color: #f8f9fa;
    color: #212529;
    padding: 8px;
    border: none;
    border-right: 1px solid #dee2e6;
    border-bottom: 1px solid #dee2e6;
    font-weight: bold;
}
QScrollBar:vertical {
    border: none;
    background: #f8f9fa;
    width: 10px;
    margin: 0px;
}
QScrollBar::handle:vertical {
    background: #dee2e6;
    min-height: 20px;
    border-radius: 5px;
}
QScrollBar::handle:vertical:hover {
    background: #adb5bd;
}
QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    height: 0px;
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: none;
}
QScrollBar:horizontal {
    border: none;
    background: #f8f9fa;
    height: 10px;
    margin: 0px;
}
QScrollBar::handle:horizontal {
    background: #dee2e6;
    min-width: 20px;
    border-radius: 5px;
}
QScrollBar::handle:horizontal:hover {
    background: #adb5bd;
}
QScrollBar::add-line:horizontal, QScrollBar::sub-line:horizontal {
    width: 0px;
}
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
    background: none;
}</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
         <property name="columnCount">
          <number>0</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="rightLayout" stretch="0,1">
       <property name="sizePolicy" stdset="0">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="target_datasource_label">
           <property name="maximumSize">
            <size>
             <width>170</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size: 16px;
font-weight: bold;
color: #000000;
background-color: transparent;</string>
           </property>
           <property name="text">
            <string>결합할 데이터 소스</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="datasource_list_combo">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>42</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox {
	background-color: white;
	border: 1px solid #dee2e6;
	border-radius: 4px;
	padding: 5px 10px;
	min-height: 30px;
	color: #495057;
}
QComboBox:focus {
	border: 1px solid #339af0;
}
QComboBox::drop-down {
	subcontrol-origin: padding;
	subcontrol-position: center right;
	width: 30px;
	border: none;
}
QComboBox::down-arrow {
	image: url(src/ui/resources/images/down-arrow-svgrepo-com.svg);
	width: 10px;
	height: 10px;
	margin-right: 1px;
}
QComboBox QAbstractItemView {
	background-color: white;
	border: 1px solid #dee2e6;
	border-radius: 4px;
	outline: none;
	selection-background-color: #e9ecef;
	selection-color: #000;
}
QComboBox QAbstractItemView::item {
	padding: 8px 20px;
	color: #495057;
}
QComboBox QAbstractItemView::item:selected {
	background-color: #e9ecef;
	color: #000;
}
</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="target_datasource_table">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>300</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QTableWidget {
    background-color: #f8f9fa;
    border: 1px solid #dee2e6;
    border-radius: 4px;
    gridline-color: #e9ecef;
    color: #212529;
}
QTableWidget::item {
    padding: 4px;
    border-bottom: 1px solid #e9ecef;
    color: #212529;
    background-color: white;
}
QTableWidget::item:selected {
    background-color: #e7f5ff;
    color: #212529;
}
QTableWidget::item:focus {
    outline: none;
    border: none;
    background: #e7f5ff;
}
QHeaderView::section {
    background-color: #f8f9fa;
    color: #212529;
    padding: 8px;
    border: none;
    border-right: 1px solid #dee2e6;
    border-bottom: 1px solid #dee2e6;
    font-weight: bold;
}
QScrollBar:vertical {
    border: none;
    background: #f8f9fa;
    width: 10px;
    margin: 0px;
}
QScrollBar::handle:vertical {
    background: #dee2e6;
    min-height: 20px;
    border-radius: 5px;
}
QScrollBar::handle:vertical:hover {
    background: #adb5bd;
}
QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    height: 0px;
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: none;
}
QScrollBar:horizontal {
    border: none;
    background: #f8f9fa;
    height: 10px;
    margin: 0px;
}
QScrollBar::handle:horizontal {
    background: #dee2e6;
    min-width: 20px;
    border-radius: 5px;
}
QScrollBar::handle:horizontal:hover {
    background: #adb5bd;
}
QScrollBar::add-line:horizontal, QScrollBar::sub-line:horizontal {
    width: 0px;
}
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
    background: none;
}</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
         <property name="columnCount">
          <number>0</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="save_button">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    color: #495057;
    background-color: #e9ecef;
    border: 1px solid #ced4da;
    padding: 8px;
    border-radius: 4px;
    font-size: 14px;
}
QPushButton:hover {
    background-color: #dee2e6;
}
QPushButton#save_button {
    color: #fff;
    background-color: #007bff;
    border-color: #007bff;
    font-weight: bold;
}
QPushButton#save_button:hover {
    color: #fff;
    background-color: #0069d9;
    border-color: #0062cc;
}
QPushButton#save_button:pressed {
    background-color: #0062cc;
    border-color: #005cbf;
}</string>
       </property>
       <property name="text">
        <string>저장</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delete_btn">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    color: #495057;
    background-color: #e9ecef;
    border: 1px solid #ced4da;
    padding: 8px;
    border-radius: 4px;
    font-size: 14px;
}
QPushButton:hover {
    background-color: #dee2e6;
}
QPushButton#save_button {
    color: #fff;
    background-color: #007bff;
    border-color: #007bff;
    font-weight: bold;
}
QPushButton#save_button:hover {
    color: #fff;
    background-color: #0069d9;
    border-color: #0062cc;
}
QPushButton#save_button:pressed {
    background-color: #0062cc;
    border-color: #005cbf;
}</string>
       </property>
       <property name="text">
        <string>삭제</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
